package.json

```diff
@@ -16,105 +16,106 @@
     "start:development:client": "node client/scripts/start.js"
   },
   "dependencies": {
-    "argon2": "^0.16.2",
```

Comment:
v0.17.0 introduce  `Major refactoring` but I tested with 0.17.3 and the authentication worked perfectly.